students/grzegorz_kowalik/lesson_02_flow_control/leap_year.py

```diff
@@ -0,0 +1,10 @@
+year = int(input())
+
+if year % 400 == 0:
```

Comment:
mozna by to poskładać w jeden warunek, zeby mieć `if-else`, zwłaszcza że aktualny podział  nie pomaga w czytelności